ecommerce/serializers.py

```diff
@@ -107,6 +118,120 @@ class Meta:
         model = models.ProductVersion
 
 
+class LineSerializer(serializers.ModelSerializer):
+    """ Line Serializer """
+
+    product = serializers.SerializerMethodField()
+
+    def get_product(self, instance):
+        """ Get the line product id"""
+        return instance.product_version.product.id
+
+    class Meta:
+        fields = ("id", "product")
+        model = models.Line
+
+
+class OrderSerializer(serializers.ModelSerializer):
+    """ Order Serializer """
+
+    name = serializers.SerializerMethodField()
+    close_date = serializers.SerializerMethodField(allow_null=True)
+    amount = serializers.SerializerMethodField()
+    discount_amount = serializers.SerializerMethodField()
+    coupon_code = serializers.SerializerMethodField(allow_null=True)
+    company = serializers.SerializerMethodField(allow_null=True)
+    line_items = serializers.SerializerMethodField()
+    b2b = serializers.SerializerMethodField()
+
+    def get_name(self, instance):
+        """ Return the deal name """
```

Comment:
`deal` -> `order` ?

Reply:
order in xpro is a deal in hubspot, the name will only be used in hubspot

Reply:
Changed to `Return the order/deal name`

Reply:
actually, that raises a separate point, so two more thoughts:

- Maybe hubspot-specific serializers should be in their own package, just so we don't accidentally incorporate them into an API by accident, it's not obvious from this class definition that I shouldn't use it in such a way. Probably worth also updating the class docstring to describe the intended usage of this serializer as well.
- Maybe rename this to `OrderToDealSerializer`?

Reply:
@rhysyngsun sounds good, but if create a hubspot app, maybe I should move the api and tasks there too?
- ecommerce
- hubspot
  - api.py
  - serializers.py
  - tasks.py

Reply:
Yeah, that makes sense to carve that all out. If it's easier to just do in a separate PR, I can approve this first though

Reply:
I went ahead and moved everything to its own hubspot app